src/client/app/components/admin/PreferencesComponent.tsx

```diff
@@ -143,6 +195,7 @@ export default function PreferencesComponent() {
 					type='text'
 					value={localAdminPref.defaultMeterReadingFrequency}
 					onChange={e => makeLocalChanges('defaultMeterReadingFrequency', e.target.value)}
+					invalid={invalidReadingFreq()}
```

Comment:
This check does not have a form feedback.

Reply:
I looked at data.ts and couldn't find a good string to let the user know that their input isn't in the right format for a duration or moment object. So for now I used error.unknown. Is this an acceptable change or would you prefer something else?

Reply:
Could you add a new key of ``invalid.input`` in all three languages that translates to "invalid input". It seems better and can be used in other places in the future.